### ISSUE TITLE
Congenitally small liver

### ISSUE THREAD

**Issue by taylordm** [obophenotype/human-phenotype-ontology#7098]
**Preferred term label:**
Congenitally small liver  (whatever term fits I'm fine with it!)

**Synonyms**
Underdeveloped or small liver (?)

**Definition (free text, please give PubMed ID)**
Congenitally small liver

**Parent term (use hpo.jax.org/app)**
HP:0410042 Abnormal Liver Morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
In Kids First, one of our children was diagnosed with a "3 cm. liver" which is a very small liver to be born with, as part of a set of congenital abnormalities.

**Your nano-attribution (ORCID)**
0000-0002-3302-4610

**Comment by pnrobinson** [comment]
@taylordm  This seems to be a very rare anomaly. Do you know if your finding can be specified, e.g., Hypoplasia of left hepatic lobe, https://pubmed.ncbi.nlm.nih.gov/28179708/
The HPO would normally use the label Hepatic hypoplasia but would include Underdeveloped liver and Congenitally small liver as synonyms.

**Comment by taylordm** [comment]
I can't find out details on this particular patient. This a third party set of phenotypes from Kids First.

**Comment by pnrobinson** [comment]
I will add a term, Segmental hypoplasia of liver. All I can find in the literature is hypoplasia of a specific lobe. 
HP:0034182
